src/main/scala/io/getclump/Sources.scala

```diff
@@ -10,70 +11,70 @@ protected[getclump] trait Sources extends Tuples {
    * the key used to get that value.
    */
   def source[KS, V, K](fetch: KS => Future[Iterable[V]])(keyExtractor: V => K)
-                      (implicit cbf: CanBuildFrom[Nothing, K, KS]): ClumpSource[K, V] =
+                      (implicit cbf: CanBuildFrom[Nothing, K, KS], ec: ExecutionContext): ClumpSource[K, V] =
```

Comment:
Do we need a separate EC for sources? Isn't it possible to use the same EC passed to `clump.get`, `clump.list` and `clump.apply`?